src/Microsoft.CodeQuality.Analyzers/UnitTests/Maintainability/AvoidDeadConditionalCode_NullAnalysis.cs

```diff
@@ -4757,16 +4788,16 @@ void M1(T t)
             return;
         }
 
-        var d = (D)t;   // Compiler error CS0030: Cannot convert type 'T' to 'D'
+        var d = {|CS0030:(D)t|};
```

Comment:
💡 It would be good to keep the comment since the method name doesn't clearly indicate that the test covers compiler error cases.

```suggestion
        var d = {|CS0030:(D)t|};   // Compiler error CS0030: Cannot convert type 'T' to 'D'
```